src/Nethermind/Nethermind.Consensus/MiningConfig.cs

```diff
@@ -70,4 +70,6 @@ public string ExtraData
             return _blocksConfig;
         }
     }
+
+    public string Signer { get; set; }
```

Comment:
Maybe it is not possible, but could we make it an `Uri?` type?

Reply:
not possible with the current JsonSerializer, but a custom one could be made.

Reply:
Maybe we should write one, is easy - here with JSON.NET: https://stackoverflow.com/a/8087049/1187056